core/src/main/java/cucumber/runtime/arquillian/ArquillianCucumber.java

```diff
@@ -69,13 +76,13 @@
 public class ArquillianCucumber extends Arquillian {
     private static final Logger LOGGER = Logger.getLogger(ArquillianCucumber.class.getName());
 
-    private static final String RUN_CUCUMBER_MTD = "____Cucumber_Runner_Not_A_Test";
```

Comment:
lets compromise with this now, I'll change the name to performCucumberOperations so it's not as tempting as runCucumberOperations, and later will dig into the runner to see how we can incorporate it into the runner (it's 6:30 am here in venezuela so I don't think i'll have the time to do it right now)